### PR TITLE
#137navigation via logo

### PR DESCRIPTION
**Pull request by BelousSofiya**
image
image
image
image

### CODE REVIEW

File: FrontEnd/src/components/cookieacception/CookieMod.module.css
Comment (Lvyshnevska) [inline FrontEnd/src/components/cookieacception/CookieMod.module.css:1]: If you remove height in .modal-content and increase margin to 14px, margin-right  to 50px in .green-btn, modal window will look better.

File: FrontEnd/src/components/SearchPage/search_page.module.css [hunk 1]
```diff
@@ -35,6 +35,7 @@
   font-weight: 600;
   color: var(--text-color);
   font: Inter;
```
[the remark this answers is not in the document]
Comment (BelousSofiya) [reply FrontEnd/src/components/SearchPage/search_page.module.css:37]: It was approved by Oleg twice: in Makar's and my PRs

File: FrontEnd/src/components/cookieacception/CookieMod.module.css
Comment (BelousSofiya) [reply FrontEnd/src/components/cookieacception/CookieMod.module.css:1]: I don't see the difference? but I believe you. Done.

File: FrontEnd/src/components/SearchPage/search_page.module.css [hunk 2]
```diff
@@ -20,7 +20,7 @@
 }
 
 .search_field_entered_value {
-  color: var(----main-style-signs-color);
+  color: #0b6c61;
   font: Inter;
```
[the remark this answers is not in the document]
Comment (Lvyshnevska) [reply FrontEnd/src/components/SearchPage/search_page.module.css:24]: I can assume that Oleg didn't notice it. 
Font is not applied correctly here. It should have mandatory values (font-size and font-family) or replaced with ordinary css property. In other case font shortland will be ignored. You can investigate it by yourself (even in dev tools of browser)

**Pull request merged**